satpy/readers/tropomi_l2.py

```diff
@@ -158,6 +165,26 @@ def _rename_dims(self, data_arr):
             dims_dict['scanline'] = 'y'
         return data_arr.rename(dims_dict)
 
+    def prepare_geo(self, bounds_data):
+        # lat/lon bounds are ordered in the following way:
```

Comment:
Could you add a docstring here and describe this in more detail? The diagram is nice, but explaining why this helps support pcolormesh would be good for future readers of the code.

Reply:
Added

Reply:
Sorry, I meant a method docstring with three quotation marks `"""` and the standard "subject line", etc. That way it will show up in the sphinx docs. You'd have to attempt to make the documentation (`cd docs; make clean; make html`) and look at the output in `build/html/`. You'll have to pay attention to errors too during the `make html`.

Reply:
I got this error when `make html`:
```
Exception occurred:
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 849, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'satpy' distribution was not found and is required by the application
```

Reply:
That suggests you didn't install Satpy.

Reply:
Em ... Actually installed. Do I need to install from the source instead of conda-forge?

Reply:
This pull request is for Satpy. You are making sure that the documentation you added in this pull request is rendered properly when documentation is made. You should be installing satpy from this pull request (`pip install --no-deps -e .`). You may need to uninstall Satpy (`conda uninstall --force satpy`) first if you installed it from conda.